static/js/admin/src/action/DeleteStreamingPropertiesResultAction.js

```diff
@@ -0,0 +1,50 @@
+define('DeleteStreamingPropertiesResultAction', [ 'jquery', 'AjaxCallbackAction'], function($, AjaxCallbackAction) {
+
+    function DeleteStreamingPropertiesResultAction(context) {
+        this._create(context);
+    };
+
+    DeleteStreamingPropertiesResultAction.prototype.constructor = DeleteStreamingPropertiesResultAction;
+    DeleteStreamingPropertiesResultAction.prototype = Object.create( AjaxCallbackAction.prototype );
+
+    DeleteStreamingPropertiesResultAction.prototype._create = function(context) {
+        this.context = context;
+        console.log(this.context)
+        var options = {
+            workMethod: "PUT",
+            workPath: `/services/api/edit/streamingProperties?id=${this.context.target.metadata.id}&url=${encodeURIComponent(this.context.target.metadata.streamingUrl)}&action=delete`,
+            workLabel: "Deleting streaming URL..",
+            followupLabel: "Deleting streaming URL...",
+            followupPath: `/services/api/status/item/${this.context.target.metadata.id}/solrRecord/version`
+        }
+
+        if ('confirm' in this.context && !this.context.confirm) {
+            options.confirm = false;
+        } else {
+            options.confirm = {
+                promptText : "Clear streaming URL?",
```

Comment:
For consistency, lets go with `Delete streaming properties?` here and down below.